DEVELOPMENT.md

```diff
@@ -419,13 +419,10 @@ To perform a release run: `git checkout master && yarn release` and follow the
 instructions.
 
 Once the release PR branch is merged into master a build will be triggered on
-Buildkite, this will build Upstream for both Linux and macOS. When the build
-has completed you can download binaries for your platform [here][ar].
+Buildkite, this will build Upstream for both Linux and macOS (unsigned).
```

Comment:
Does it make sense to rename this section to “Preparing a release”? Just “Release” makes it sound like it is end-to-end.

Reply:
https://github.com/radicle-dev/radicle-upstream/pull/1406/commits/354b7e5bae9dd34e141c560ba92e32672b3cdd10